docs/contents/releases.md

```diff
@@ -1,128 +1,159 @@
-  available via a public container registry
+ACK controllers in the `General Availability` Maintenance Phase may move to a
+`Deprecated` Maintenance Phase after a Deprecation Warning notice has been sent
+out (and the controller's documentation has been updated with said deprecation
+notice).
 
-The Custom Resource Definition (CRD) `APIVersion` for resources managed by ACK
-service controllers in `BETA` will carry an `beta` designation -- e.g.
-`v1beta1`. Developers can expect minor changes to the structure and format of
-the CRDs during the `BETA` release status, however **any change to the CRD
-format during the `BETA` release status will result in an incremented
-`APIVersion` for the CRD**. For example, consider a CRD with a
-`GroupVersionKind` (GVK) of `Bucket.s3.services.k8s.aws/v1beta2`. If the format
-of that CRD changed, we guarantee that a new `v1beta3` package and
-corresponding `Bucket.s3.services.k8s.aws/v1beta3` GVK would be released,
-allowing developers to cleanly migrate between API versions of their custom
-resources.
+Controllers in `Deprecated` Maintenance Phase continue to receive the same
+level of support as controllers in the `General Availability` phase.
 
-## GENERALLY AVAILABLE
+### Not Supported
 
-An ACK service controller reaches the `GENERALLY AVAILABLE` (GA) release status
-once a controller in the `BETA` release status has satisfied the following
-criteria:
+ACK controllers may eventually be moved into a `Not Supported` Maintenance Phase.
 
-* End-to-end test coverage of all CRUD operations for all service API
-  resources for the specific service
-* Test coverage for "negative" or "fuzz" testing to ensure validating webhooks
-  properly guard the resource creation and mutation
-* Long-running "soak" testing to ensure controller reliability and stability
-* Documentation is included that demonstrates usage of the custom resources
-  managed by the controller
-
-The Custom Resource Definition (CRD) `APIVersion` for resources managed by ACK
-service controllers in `GENERALLY AVAILABLE` will have a major version with no
-`alpha` or `beta` designation -- e.g.  `v1`. Developers can expect no changes
-to the structure and format of the CRDs once the controller is in the
-`GENERALLY AVAILABLE` release status.
-
-!!! note "How long from developer preview to GA?"
-    We aim to get an ACK service controller into the `GENERALLY AVAILABLE`
-    release status within 3 months of when the controller is released as
-    `DEVELOPER PREVIEW`. Some controllers will naturally take longer due to
-    inconsistencies, corner cases or complexity of the underlying AWS service
-    API.
+A controller major version series may move from the `Preview` Maintenance Phase
+to the `Not Supported` Maintenance Phase at any time. This may happen if the
+team maintaining the controller determines it is not possible to get the
+controller with that major version series into a `General Availability` phase.
```

Comment:
would we not transit a deprecated phase here?

Reply:
No. We don't deprecate Preview stuff.